lsp-ui-flycheck.el

```diff
@@ -2,7 +2,9 @@
 
 ;; Copyright (C) 2017  fmdkdd
 ;; URL: https://github.com/emacs-lsp/lsp-ui
-;; Keywords: lsp, ui
+;; Keywords: languagues, tools
+;; Package-Requires: ((emacs "26.1") (dash "2.14") (dash-functional "1.2.0") (lsp-mode "6.0"))
```

Comment:
Why add the requirements to each file ?

Reply:
I tried to add [package-lint](https://github.com/purcell/package-lint), but for other problems that was not so easy to fix, I gave up, I'll undo this